pkg/plugin/clientmgmt/restoreitemaction/v1/shared_test.go

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
shared_test.go file is same with backupitemaction's.
Can we let all plugins use the same shared_test.go file?

Reply:
@blackpiglet unfortunately, the way the test framework works, we can't, at least the way it's currently implemented. _test files aren't exported in the package. We could create a util package somewhere and use that, although then could end up with test framework-related code in the main velero image. That may be OK, though. Basically if we want this to remain part of the tests only, then copying it into the varius packages will be needed. If we're ok making it a shared, exportable package, then we could refactor useage here (and in the other places it's used).

Reply:
@blackpiglet ok, I went ahead and made the changes. This code is now in internal/restartabletest and I've removed all copies of it (the one in this PR and the two already merged) and updated references to use the exported package instead.